src/cst1201/Simulator.java

```diff
@@ -0,0 +1,11 @@
+package cst1201;
+
+public class Simulator {
+
+    public static void main(String[] agrs) {
+        Bowtie turtle = new Bowtie();

```

Comment:
`turtle` probably isn't a good name for a `Bowtie` object reference. Perhaps `bowtie` would be better.